pkg/util/namespace/namespace.go

```diff
@@ -15,6 +15,7 @@ func IsOpenShiftNamespace(ns string) bool {
 		"openshift-azure-logging":            {},
 		"openshift-azure-operator":           {},
 		"openshift-managed-upgrade-operator": {},
+		"openshift-azure-guardrails":         {},
```

Comment:
please consider adding this namespace to the test file as it is critical for us to monitor as well.